lib/components/popups/course-search/search-results/Placeholder.tsx

```diff
@@ -342,7 +342,7 @@ const Placeholder: FC<{ addCourse: (plan?: Plan) => void }> = (props) => {
             </div>
           </div>
           <Select
-            options={['None', 'N', 'S', 'H', 'E', 'Q'].map((area: any) => ({
+            options={['none', 'N', 'S', 'H', 'E', 'Q'].map((area: any) => ({
```

Comment:
ok if you have time, i'd double check if in the backend we do use 'None' and not 'none' 

Pretty sure you should keep as None, unless you want to make changes in db and backend 

<img width="1201" alt="Screenshot 2023-06-14 at 8 52 30 PM" src="https://github.com/uCredit-Dev/ucredit_frontend_typescript/assets/90944895/75b8ba3c-8222-4f19-ac96-f57604ec0008">

Reply:
smoge why inconsistent :( oke will do yes ma'am

Reply:
hoping you tested + nothing breaks